File: Problem 9/ans.py
for c in range(1,1001):
    for b in range(1,c):
        for a in range(1,b):
            if ((a*a+b*b)==(c*c)) and (a+b+c==1000):
                print("Product: %d, a: %d, b: %d, c:%d\n" % (a*b*c,a,b,c))
                break

File: Problem 9/ans.cpp
#include<iostream>

using namespace std;

int main()
{
    for(int c=0; c<=1000; c++)
        for(int b=0; b<c; b++)
            for(int a=0; a<b; a++)
                if (a+b+c==1000 && c*c==a*a+b*b)
                    cout << "Product: " << a*b*c << ", a: " << a << ", b: " << b << ", c: " << c << endl;
    return 0;
}
